### DISCUSSION TITLE
Error "You exceeded your current quota, please check your plan and billing details"

### DISCUSSION THREAD

**Discussion by supwang** [ChatGPTNextWeb/NextChat#1906] [Q&A]
Hi,

What means of below error?

{
  "error": {
    "message": "You exceeded your current quota, please check your plan and billing details.",
    "type": "insufficient_quota",
    "param": null,
    "code": null
  }
}

Thank you.

**Comment by Gavin-Meskin-Chen** [comment]
I also reported this error, and I still have this error after updating the API key.

**Reply by jasonDwq** [reply]
有伙伴解决了吗

**Reply by supwang** [reply]
只有成为付费用户才能正常使用了
